packages/rke2-runtimeclasses/charts/values.yaml

```
@@ -0,0 +1,7 @@
+runtimeClass:
```

Nit: should this be plural?
```suggestion
runtimeClasses:
```